### PR TITLE
16 January/2024 ✅ Editor updated

### PR DESCRIPTION
**Pull request by asliddintursunov**
Suddenly deleted all the last 3 day changes in Fork and Master and Wrote them again. Editor is updated, HeaderContent component is created and heading content is put there. Removed dark/light mode.

### CODE REVIEW

File: components/Editor.tsx [hunk 1]
```diff
@@ -1,57 +1,28 @@
...
-    if (match) {
-      const pic = match[1];
-      setArticle({ pic });
-      setPic(pic);
-      console.log(match[1]);
-    } else {
-      console.log("no match");
-    }
   };
   const handleClear = function () {
```
Comment (asliddintursunov) [inline components/Editor.tsx:16]:
```suggestion
  const handleClear = function () {}
```

File: app/api/getTrendingsByCategory/route.ts [hunk 2]
```diff
@@ -1,7 +1,14 @@
-import { NextResponse } from "next/server";
+import { NextResponse, NextRequest } from "next/server";
 const trendings_by_category = require("@/database/trending_by_category.json");
-export async function GET(response: Response) {
+const articles_by_category = require("@/database/atricles_by_category.json");
+export async function GET(req: NextRequest, res: NextResponse) {
+  const category = req.nextUrl.searchParams.get("category");
+  if (category) {
+    console.log("category ->", category);
```
Comment (muslimchikkk) [inline app/api/getTrendingsByCategory/route.ts]: Let's remove this

File: components/RelatedArticlesButton.tsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+type Props = {
+  category: string;
+};
+
+function RelatedArticlesButton({ category }: Props) {
+  const handleClick = async function (ca: string) {
+    const data = ca.split(" ").join("").toLowerCase();
+    const res = await getRelatedArticles(data);
+    console.log(res);
```
Comment (muslimchikkk) [inline components/RelatedArticlesButton.tsx]: and this

File: components/RelatedArticlesButton.tsx [hunk 4]
```diff
@@ -0,0 +1,25 @@
+"use client";
+
+import getRelatedArticles from "@/lib/getRelatedArticles";
+
+type Props = {
+  category: string;
+};
+
+function RelatedArticlesButton({ category }: Props) {
+  const handleClick = async function (ca: string) {
```
Comment (muslimchikkk) [inline components/RelatedArticlesButton.tsx]: What is ca? a good naming helps us to make the code readable :)

**Review by muslimchikkk — APPROVED** [review]

File: components/RelatedArticlesButton.tsx (hunk 4 above)
Comment (asliddintursunov) [reply components/RelatedArticlesButton.tsx]: Sorry, I was just testing in that moment, that's why I name it so. I will change it :)

**Pull request merged**